src/script.js

```diff
@@ -5,3 +5,29 @@ La apăsarea unui buton (.buttons a):
     - Adaugă clasa "active" la butonul apăsat
     - Adaugă clasa "active" la tab-ul identificat prin atributul data-target
 */
+const buttons = document.querySelectorAll(".buttons a");
+const articles = document.querySelectorAll(".tabs article");
+buttons[0].addEventListener("click", () => {
+  buttons[0].classList.add("active");
+  buttons[1].classList.remove("active");
+  buttons[2].classList.remove("active");
+  articles[0].classList.add("active");
+  articles[1].classList.remove("active");
+  articles[2].classList.remove("active");
+});
+buttons[1].addEventListener("click", () => {
+  buttons[0].classList.remove("active");
+  buttons[1].classList.add("active");
+  buttons[2].classList.remove("active");
+  articles[0].classList.remove("active");
+  articles[1].classList.add("active");
+  articles[2].classList.remove("active");
+});
+buttons[2].addEventListener("click", () => {
+  buttons[0].classList.remove("active");
+  buttons[1].classList.remove("active");
+  buttons[2].classList.add("active");
+  articles[0].classList.remove("active");
+  articles[1].classList.remove("active");
+  articles[2].classList.add("active");
+});
```

Comment:
O problemă de programare de obicei poate fi rezolvată în mai multe feluri.

Ideea ta nu e neapărat rea, dar ai putea să o mai optimizezi.

E bine să gândești generalist: acolo unde sunt 3 butoane, pot fi și 2, sau 4, sau 50.   
Ai putea scrie ceva care să acopere toate cazurile posibile.   
Pentru că trebuie să „vizitezi” fiecare buton, poți folosi `for` sau `buttons.forEach`.